docker/README.md

```diff
@@ -1,12 +1,19 @@
-# Docker
+# Requirements
 
-Since fylr requires a fully running installation of docker, refer to it's official documentation [how to install docker](https://docs.docker.com/engine/install/) and follow these steps.
+* Since fylr requires a fully running installation of docker, refer to it's official documentation [how to install docker](https://docs.docker.com/engine/install/) and follow these steps.
+* 16 GB of RAM
```

Comment:
We don't need 16gb of memory. In our cloud infrastructure, a single deployment of fylr has just a maximum of 4gb. Only migration install types get a 16gb memory setting.

On top of that, elasticsearch needs 2gb (at least that's what we configured in docker-compose) and additional memory for postgres and minio. Overall, I would say we can easily work with 8gb of memory.

Reply:
Let's keep it like that.